static/bidder-info/edge226.yaml

```diff
@@ -0,0 +1,20 @@
+endpoint: "http://ssp.dauup.com/pserver"
+maintainer:
+  email: "audit@edge226.com "
```

Comment:
@Edge226Ads We have sent a verification mail to `audit@edge226.com`. Please reply to that mail with "Received" to complete the verification process.
Thank you!

Reply:
Verification completed!